### PR TITLE
TriggerAuthentication/Vault: add support for vault namespace

### PR DESCRIPTION
**Pull request by chapurlatn**
Signed-off-by: Nicolas Chapurlat <nc@coorganix.com>

### Changes

Related to keda PR [https://github.com/kedacore/keda/pull/2085](https://github.com/kedacore/keda/pull/2085)
- Just add the optional Vault namespace in the Vault TriggerAuthentication example.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/2084

### CODE REVIEW

File: content/docs/1.5/concepts/authentication.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ spec:
     containerName: {container-name} # Optional. Default: scaleTargetRef.containerName of ScaledObject
   hashiCorpVault: # Optional.
     address: {hashicorp-vault-address} # Required.
+    namespace: {hashicorp-vault-namespace} # Optional. Useful for Vault Enterprise
```
[the remark this answers is not in the document]
Comment (tomkerkhove) [reply content/docs/1.5/concepts/authentication.md:105]: Next to that, can you update the "Hashicorp Vault secret(s)" section below as well please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Thank you! It will just need a bit more changes.

File: content/docs/1.5/concepts/authentication.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chapurlatn) [reply content/docs/1.5/concepts/authentication.md:105]: k, I'll do that (sorry for the bad version..)

[the remark this answers is not in the document]
Comment (chapurlatn) [reply content/docs/1.5/concepts/authentication.md]: done, with a lightweight comment about `namespace` in description

**Review by tomkerkhove — APPROVED** [review]
Thank you!

**Review by zroubalik — APPROVED** [review]

**Pull request merged**